kernel/src/fs/cgroupfs/systree_node.rs

```diff
@@ -496,9 +608,47 @@ inherit_sys_branch_node!(CgroupNode, fields, {
 
                 Ok(len)
             }
+            "cgroup.subtree_control" => {
+                let (actions, len) = read_subtree_control_from_reader(reader)?;
+
+                // The Lock order: parent controller -> current controller -> child controllers
+                let parent_node = self.cgroup_parent().unwrap();
```

Comment:
This `unwrap` may fail if the directory and its parent has been removed? Note that someone can hold a file descriptor to a deleted directory.

Check other uses of `cgroup_parent` as well.